src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderRepository.kt

```diff
@@ -43,6 +43,21 @@ interface OffenderRepository :
     """,
   )
   fun findAllIds(pageable: Pageable): Page<PrisonerId>
+
+  @Query(
+    """
```

Comment:
I had to specify the index as otherwise it uses the root offender index, gets the order wrong, and is as slow as the spring paged version!

Reply:
I am a bit confused' wouldn't you just order by root_offender_id which has an index?

Reply:
Should it just be 

`
 select offender_id_display, root_offender_id from Offenders where root_offender_id = offender_id order by root_offender_id
`

Reply:
This doesnt work either - it is slow and the ordering is still wrong , not sure why .